packages/fxa-shared/l10n/branding.ftl

```diff
@@ -25,7 +25,12 @@
 -product-mozilla-account = Mozilla account
 
 # "accounts" can and should be localized, "Mozilla" must be treated as a brand. Plural "Mozilla accounts" is used when referring to something affecting all Mozilla accounts, not just the individual's account.
--product-mozilla-accounts = Mozilla accounts
+# uppercase variant is primaly used for Title case, and legal contexts
+-product-mozilla-accounts =
```

Comment:
I've added a capitalization option for the rare case where we need to uppercase "Mozilla Accounts", with "Mozilla accounts" as default.

@bcolsson I'm realizing now that this might need a new id though since the string has already been extracted... Not sure if that is best, or if I should add a new uppercase term instead in addition to the existing one.

Reply:
I gave this some thought, I think we'll want to parameterize the existing term. Since this change is mostly a capitalization change which is locale independent it doesn't warrant a new ID. This also creates potentially redundant terms for many locales that have different scripts or styles. 

When we do add the parameter to the term, I'll send a message to locales about the update to the term just to cover our bases here.